### PR TITLE
Fix #1275

### LINKED ISSUE #1275: update modules in pipeline template
Merging https://github.com/nf-core/tools/pull/1247 broke a few tests on `dev` which we can fix by reinstalling the modules into the pipeline template, once they have been updated to emit a `versions.yml` in the modules repository (see also https://github.com/nf-core/modules/pull/665)

### PR DESCRIPTION
**Pull request by drpatelh**
Get tests passing again after changing the way we report software versions from each module.

### CODE REVIEW

File: nf_core/pipeline-template/conf/base.config [hunk 1]
```diff
@@ -54,4 +54,7 @@ process {
         errorStrategy = 'retry'
         maxRetries    = 2
     }
+    withName:CUSTOM_DUMPSOFTWAREVERSIONS {
+        cache = false
```
Comment (grst) [inline nf_core/pipeline-template/conf/base.config:58]: Why is this necessary? (I know it was like that before, but i never understood why)

File: nf_core/pipeline-template/modules/nf-core/modules/custom/dumpsoftwareversions/main.nf [hunk 2]
```diff
@@ -91,5 +93,13 @@ process GET_SOFTWARE_VERSIONS {
         yaml.dump(versions, f, default_flow_style=False)
     with open("software_versions_mqc.yml", 'w') as f:
         yaml.dump(versions_mqc, f, default_flow_style=False)
+
+    yaml_version = {}
+    yaml_version["${getProcessName(task.process)}"] = {
```
Comment (grst) [inline nf_core/pipeline-template/modules/nf-core/modules/custom/dumpsoftwareversions/main.nf:98]: This is probably just to make the pytests happy .. but if we have it anyway, should we also include it into the yaml containing the collected versions?

File: nf_core/pipeline-template/conf/base.config (hunk 1 above)
Comment (drpatelh) [reply nf_core/pipeline-template/conf/base.config:58]: So MultiQC is always re-run regardless of whether the pipeline is cached or not.

Comment (drpatelh) [reply nf_core/pipeline-template/conf/base.config:58]: I think there were issues around it not being re-run properly when it had to be but not 💯

File: nf_core/pipeline-template/modules/nf-core/modules/custom/dumpsoftwareversions/main.nf (hunk 2 above)
Comment (drpatelh) [reply nf_core/pipeline-template/modules/nf-core/modules/custom/dumpsoftwareversions/main.nf:98]: Good idea 👍🏽

Comment (drpatelh) [reply nf_core/pipeline-template/modules/nf-core/modules/custom/dumpsoftwareversions/main.nf:98]: Added in https://github.com/nf-core/modules/pull/764 and will re-install here once that has been merged.

**Review by grst — APPROVED** [review]
LGTM!
